### PR TITLE
Fixed documentation issues over new documentation structure

### PR DESCRIPTION
**Pull request by rr-santos**
Currently, documentation had multiple opened issues with existing content.

Added multiple commits that individually fix issues from #23 to #35 using the new documentation tree structure.

### CODE REVIEW

File: getting_started/configure_baseboxd.md [hunk 1]
```diff
@@ -63,89 +63,41 @@ uname -a
 
 ### Configure a local or remote controller
 
-BISDN Linux contains the prerequisites to control the switch by either local or remote OpenFlow controllers. The default configuration is a local controller.
-Run the following scripts in the BISDN Linux shell to configure the local or remote usage.
+BISDN Linux contains the prerequisites to control the switch by either local or remote OpenFlow controllers. The default configuration is a local controller, with BISDN Linux currently supporting baseboxd and [Ryu](https://osrg.github.io/ryu/).
```
Comment (jklare) [inline getting_started/configure_baseboxd.md:66]:
```suggestion
BISDN Linux contains the prerequisites to control the switch by either local or remote OpenFlow controllers. The default configuration is a local controller, with BISDN Linux currently supporting [baseboxd](https://github.com/bisdn/basebox) and [Ryu](https://osrg.github.io/ryu/).
```

File: getting_started/connecting_to_the_switch.md [hunk 2]
```diff
@@ -6,12 +6,23 @@ nav_order: 1
...
-Example of .kermrc:
+Here, we provide instructions for [minicom](https://linux.die.net/man/1/minicom) and [ckermit](http://www.columbia.edu/kermit/ckututor.html), assuming the switch is connected in port `/dev/ttyUSB0`.
+
+
+### minicom
+
+The default configurations of minicom can be used to connect to the switch, with exception of the console port. Run minicom in a command line with the following command:
+
+```
+sudo minicom -D /dev/ttyUSB0
```
Comment (jklare) [inline getting_started/connecting_to_the_switch.md:19]: i think we should mention here that you usually need to disable "FlowControl" in the serial port configuration of minicom

File: getting_started/install_bisdn_linux.md [hunk 3]
```diff
@@ -29,6 +29,8 @@ to install a switch image. To remove the image, enter
 run onie_uninstall
 ```
 
+The BISDN image can be obtained either through the [ONIE CLI](#get-the-image-via-the-cli) or through a [DHCP server](#get-the-image-via-dhcp-option-60). Both methods are described below.
```
Comment (jklare) [inline getting_started/install_bisdn_linux.md:32]:
```suggestion
The BISDN image can be installed either via the [ONIE CLI](#get-the-image-via-the-cli) or through a [DHCP server](#get-the-image-via-dhcp-option-60). Both methods are described below.
```

File: getting_started/connecting_to_the_switch.md (hunk 2 above)
Comment (rr-santos) [reply getting_started/connecting_to_the_switch.md]: Updated commit df33c58d842394a9546bd3559a2c4489bbd7a2ee with these changes

**Comment by KanjiMonster** [comment]
>  that individually fix issues from #23 to #35 

Your commits only reference #23 to #28 and #35, but not #29 to #34 - please also tag these issues or update the description.

Also #39 also intends to fix #28, will these PRs conflict?

**Comment by jklare** [comment]
lgtm, thanks

**Pull request merged**